.env

```
@@ -0,0 +1,10 @@
+API_TOKEN=
+DB_PASSWORD=hd121612
+DB_URL=jdbc:postgresql://cpdb.cf6k82ssc832.ap-northeast-2.rds.amazonaws.com:5432/cpdb
+DB_USERNAME=postgres
+JWT_SECRET=J25P6nDtgArp8fljXAwPiiQaQvc6N0or
+R2_ACCESS_KEY=5abfeb699aefbae399603640cbd75ff3
+R2_BUCKET=cp-main-be
+R2_ENDPOINT=https://a35471a7db408765c522b98a7b65d786.r2.cloudflarestorage.com/cp-main-be
+R2_SECRET_KEY=72834ca03a49383c5893e3d567ecb1a17de78ab1c21dd73e871ce8a11216bfd1
+FASTAPI_URL=http://13.54.125.183:8000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FASTAPI_URL key should go before the JWT_SECRET key

(UnorderedKey)